site/src/badgesInfo.json

```diff
@@ -109,6 +109,12 @@
             "location": "assets/badges/continuous-contributor/continuous-contributor.png",
             "title": "Continuous Contributor",
             "description": "This badge is awarded to the community members who make consistent and impactful contributions for a long period of time in Layer5 projects in recognition and appreciation of their efforts."
+        },
+        {
+            "slug": "hacktoberfest-contributor",
```

Comment:
Should this be hacktoberfest-23-contributor? I suppose if it was, Cloud would need to be updated, too.